### PR TITLE
Mode some modification on the ODS and NDWH scripts resulting from the…

### PR DESCRIPTION
**Pull request by DennisGibz**
… last completed refresh

### CODE REVIEW

**Review by cbrianbet — APPROVED** [review]

File: Scripts/ODS/MNCH DOCKET/Load_MNCH_Heis.sql [hunk 1]
```diff
@@ -36,11 +36,10 @@ BEGIN
...
-									P.SiteCode,  
-									[DNAPCR1Date],HEIExitCritearia,
+									P.SiteCode,
 					
-					ROW_NUMBER() OVER (PARTITION BY P.PatientPK,P.SiteCode,HEIExitCritearia
-					ORDER BY P.PatientPK,P.SiteCode,[DNAPCR1Date]) Row_Num
+					ROW_NUMBER() OVER (PARTITION BY P.SiteCode,P.PatientPK
+					ORDER BY P.PatientPK,P.SiteCode) Row_Num
 					FROM [ODS].[dbo].[MNCH_Heis] p
 					where HEIExitCritearia like '%Confirmed HIV Positive%')
```
Comment (nobert-mumo) [inline Scripts/ODS/MNCH DOCKET/Load_MNCH_Heis.sql:44]: @DennisGibz was wondering what the logic behind was where clause?

File: Scripts/ODS/MNCH DOCKET/Load_MNCH_Heis.sql [hunk 2]
```diff
@@ -36,11 +36,10 @@ BEGIN
 
 					;with cte AS ( Select            
 									P.PatientPK,            
-									P.SiteCode,  
-									[DNAPCR1Date],HEIExitCritearia,
+									P.SiteCode,
 					
-					ROW_NUMBER() OVER (PARTITION BY P.PatientPK,P.SiteCode,HEIExitCritearia
```
Comment (nobert-mumo) [inline Scripts/ODS/MNCH DOCKET/Load_MNCH_Heis.sql:42]: @DennisGibz does it mean from the partition that was checking fro dups it was possible for a HEI to have two more than one `HEIExitCritearia`?

**Review by nobert-mumo — CHANGES_REQUESTED** [review]

File: Scripts/ODS/CT_DOCKET/CT_PatientPharmacy.sql [hunk 3]
```diff
@@ -34,7 +34,7 @@ BEGIN
 			VALUES(@DispenseDate,GETDATE())
 
 			MERGE [ODS].[dbo].[CT_PatientPharmacy] AS a
-				USING(SELECT Distinct top 20
```
[the remark this answers is not in the document]
Comment (DennisGibz) [reply Scripts/ODS/CT_DOCKET/CT_PatientPharmacy.sql:37]: Top 20 manenos. Kudos to my previous reviewers @nobert-mumo  and @cbrianbet

File: Scripts/ODS/MNCH DOCKET/Load_MNCH_Heis.sql (hunk 2 above)
Comment (DennisGibz) [reply Scripts/ODS/MNCH DOCKET/Load_MNCH_Heis.sql:42]: I think it was a data quality related problem that we were trying to address. I don't think where clause servers the purpose anymore.

Comment (DennisGibz) [reply Scripts/ODS/MNCH DOCKET/Load_MNCH_Heis.sql:42]: tested by set removing the where clause and it has no impact on the result.

File: Scripts/NDWH/ALL DIMENSIONS/load_DimHTSTraceOutcome.sql [hunk 4]
```diff
@@ -26,3 +26,15 @@ MERGE [NDWH].[dbo].[DimHTSTraceOutcome] AS a
 								END 
 		FROM [NDWH].[dbo].[DimHTSTraceOutcome] source_data;
 
+		with cte AS (
+						Select
```
[the remark this answers is not in the document]
Comment (DennisGibz) [reply Scripts/NDWH/ALL DIMENSIONS/load_DimHTSTraceOutcome.sql:30]: Added TracingOutcome IS NOT NULL to deal the dulpicates. However,we will continue monitoring using the metabase dashboard.

**Comment by Marymary-dev** [comment]
Requesting visibility on this work on ODS @koskedk

**Review by nobert-mumo — APPROVED** [review]
LGTM

**Pull request merged**